specs/requests/artifacts_spec.py

```diff
@@ -133,6 +133,73 @@
                 with it('returns a 422 status code'):
                     expect(self.response.status_code).to(equal(422))
 
+        with description('UPDATE MANY'):
+            with context('valid request'):
+                with context("all the resources exist"):
+                    with before.each:
+                        with Mock() as elastic_mock:
+                            for i in range(2):
+                                elastic_mock.get(
+                                    doc_type='_all', id=f'{get_uuid(i)}',
+                                    index='artifact').returns(es_get_response(i))
+                            elastic_mock.update(
+                                doc_type='image', id=f'{get_uuid(0)}', index='artifact',
+                                body={'doc': {
+                                    "updated_at": matches_regexp(date_regex()),
+                                    "tags": ["blue", "red"]}})
+                            elastic_mock.update(
+                                doc_type='image', id=f'{get_uuid(1)}', index='artifact',
+                                body={'doc': {
+                                    "updated_at": matches_regexp(date_regex()),
+                                    "tags": ["blue", "green"]}})
+
+                        current_app.es = elastic_mock
+                        self.response = self.context.client().patch(f"/images", json={
+                            "data": [
+                                {
+                                    "id": get_uuid(0),
+                                    "tags": ["blue", "red"]
+                                },
+                                {
+                                    "id": get_uuid(1),
+                                    "tags": ["blue", "green"]
+                                }
+                            ]
+                        })
+
+                    with it('returns a 204 status code'):
+                        expect(self.response.status_code).to(equal(204))
```

Comment:
I think this deserves some more specific Mock expects with doublex-expects. https://pypi.org/project/doublex-expects/
Simply adding: `expect(current_app.es).to(have_been_satisifed)` would be sufficient I think.